saleor/graphql/attribute/tests/mutations/test_attribute_create.py

```diff
@@ -52,10 +53,11 @@ def test_create_attribute_and_attribute_values(
 
     attribute_name = "Example name"
     name = "Value name"
+    value = "Value"
     variables = {
         "input": {
             "name": attribute_name,
-            "values": [{"name": name}],
+            "values": [{"name": name, "value": value}],
             "type": AttributeTypeEnum.PRODUCT_TYPE.name,
```

Comment:
IMO, we should also have tests for defining `values` without given `value` in the `values` input field.

Reply:
`test_create_attribute_with_file_input_type_and_values`
`test_create_attribute_with_file_input_type_and_values` 
Already does that 👌